src/main/java/org/jenkinsci/maven/plugins/hpi/RunMojo.java

```diff
@@ -161,7 +163,7 @@ public class RunMojo extends AbstractJettyMojo {
      *
      * If connectors are configured in the Mojo, that'll take precedence.
      */
-    @Parameter(property = "port")
+    @Parameter(property = "port", defaultValue = "8080")
     protected int defaultPort;
```

Comment:
Jetty's behavior changed to give us a random port every time. To preserve the existing behavior (which defaults to port 8080) I had to add this. In the future, we could enhance HPI plugin to work with Jetty's new random port behavior, but the logic to print the URL would need to be adjusted. Since the goal of this PR was to do the minimum amount of change necessary to complete the upgrade, I did not tackle that in this PR.

Reply:
I think we would prefer to keep the static port anyway, since this is for interactive use and having a stable root URL across runs is desirable.

Reply:
Yeah, I felt the same way after having a page open in my browser and wanting to refresh it and get an updated version without having to change the port.